src/events/aws_sdk_v3.js

```diff
@@ -79,11 +79,17 @@ function getOperationByCommand(command) {
  * @param {Function} wrappedFunction The function to wrap
  * @returns {Function} The wrapped function
  */
-function AWSSDKv3WrapperSNS(wrappedFunction) {
-    return function internalAWSSDKv3WrapperSNS(command) {
+function AWSSDKv3Wrapper(wrappedFunction) {
+    return function internalAWSSDKv3Wrapper(command) {
         try {
-            const serviceIdentifier = this.config.serviceId.toLowerCase();// sns
+            const serviceIdentifier = this.config.serviceId.toLowerCase();
             const resourceName = '';
+
+            if (!(serviceIdentifier in specificEventCreators)) {
```

Comment:
this might be my personal taste but I feel that `if (!specificEventCreators.includes(serviceIdentifier)` is clearer. The specificEventsCreators need to be an array for that case. Also I would consider migrating the specificEventsCreators const to the consts file (maybe rename as well to expose it's SDK v3 events, say supportedSDKV3Events)

Reply:
It is the same as in v2. This is why I prefer to keep it like that.

Reply:
I would prefer having our supported events on a single js file instead of browsing each instrumentation